### ISSUE TITLE
First import

### ISSUE THREAD

**Issue by OSAwebsite** [code4recovery/12-step-meeting-list#166]
I am trying to demonstrate to another party the ease of use of the plugin. As I am preparing, I have a full csv file of all of my own meetings. The csv file is set up the way I use it for my own website. I can see that everything was nicely imported but when I view the meeting details page, it does not show a lot of into. The meeting list comes up with 'page not found'. Not the way I expected it.

I have no outstanding updates for plugins or themes, I use the same set up as my live website and by the looks of it, everything is where it should be. Addresses are valid and all that. Where should I look?

The spot where I see just a little bit of info is when I click 'Where is my info'.

My staging site is here:
http://www.iamnow.today/meetings/

And the link in the main nav menu 'test Meetings' points to 'http://www.iamnow.today/meetings/

What am I missing?

This site will not stay public by the way, so if you find it under construction let me know.

thanks,
Nicky

**Comment by OSAwebsite** [comment]
I just deactivated, deleted and re-installed the plugin. No dice. 
The meeting list does not show up and the details page is just like a summary.
When I perform a search for the meeting I do find it though. But the standard link to the meeting list does not want to work.

**Comment by tech2serve** [comment]
@OSAwebsite,

Did you try going into the Permalinks page and clicking `save`? That usually refreshes the links and often resolves issues like this. 

On your staging site, have you entered an API key for mapping?

**Comment by OSAwebsite** [comment]
Have just reset it to plain, it was on custom, although I do not know why. I usually don't touch these things.
Now it says it cannot find the page, error 404.

I have no API key for mapping.

**Comment by tech2serve** [comment]
You still need an api key so the meeting can geocode and be published.

You shouldn't need to change the permalink settings so much as clicking `save` which triggers a refresh in Wordpress.

**Comment by OSAwebsite** [comment]
I don't have one in my live site so I figured I'd leave it empty.

Permalinks: so, it was on custom, now that I changed to plain have I lost something? I did save it. What now?

**Comment by OSAwebsite** [comment]
This is strange. In Chrome, where I am logged in as the administrator I see the list of meetings now. But in Safari, when I refresh the front end stil get the 404 error. For you understanding: on Chrome I now see the full page including the map location. The link of the page seems to be: http://www.iamnow.today/?post_type=tsml_meeting
Is this correct? Is that what I should connect the menu to?

**Comment by OSAwebsite** [comment]
OK, now Safari works after I quite it and reopened. Must be a cache thing. Frightening!

**Comment by tech2serve** [comment]
You don't have an API key on your live site? Not sure how it would work then? What is the link to your live site?

Go to your admin page-->Meetings-->Settings, and look down towards the bottom right side of the page. One of these two fields needs to be filled in:

<img width="285" alt="image" src="https://user-images.githubusercontent.com/31904316/93006910-5a89e800-f51f-11ea-9e10-3f07fa99b8c1.png">

**Comment by OSAwebsite** [comment]
Yeah, the upper one is filled. So that explains that.

**Comment by OSAwebsite** [comment]
I thought you wanted me to look at just the API Key for Google. I am using Mapbox instead. I don't remember when, but there was a good reason.

**Comment by OSAwebsite** [comment]
Now, if you would, can you explain about permalinks?
I know what the word means, but I cannot remember changing it in my live site. There it is set to Post name.
It might be that I changed that, like I said, I just don't recall.

But in earlier issues with TSML I saw mentions of the permalinks as well. I am not sure I understand what the significance of them is, in terms of how a fresh new webpage with new content can have issues with the permalink. Or is this too complex to explain?

**Comment by tech2serve** [comment]
> OK, now Safari works after I quite it and reopened. Must be a cache thing. Frightening!

Just has to be one or the other. They are both technically API keys, Mapbox just calls theirs a "token". My apologies for that confusion.

Cache is the other common issue of data not showing up properly. When I first went to your staging site, I didn't see the meetings either. That tells me the issue probably wasn't your browser, but rather your website cache which probably took awhile to update.

Permalinks have less to do with TSML and more to do with Wordpress. Not sure if you ever changed them; it really shouldn't matter what you use, but I generally don't change them without good reason. Each permalink tells Wordpress where to find the page, post, and images. They are also used to point to our meetings, locations, groups, etc. I'm not sure why a new site would need them refreshed, or if they really did needed to be updated (was the problem really just with the cache?). All I know is that Wordpress users seem more likely to have problems with them when they do something that changes website URLs; and this is more common with staging sites.

**Comment by OSAwebsite** [comment]
Thanks so much for that explanation - it makes more sense to me now.
Well.... I did have the plug-in installed before, for testing. I have another outstanding issue (not sure you were the one communication to me about that) and the person asked me to remove everything, then take a clone of a newer version for testing. When I got to getting the clone, the page was not found on Github. So then I had this other party who I am meeting with tomorrow and wanted to have something in place to show them. This is why I re-installed the current version, then did the import. Oh well. Cache can be a real B....

The thing that I did notice was that after the re-install I saw the deleted items in the trash and tried to delete them. Did not work. The trash might be empty already, just not showing it also due to cache.

Well, it seems solved so for now I will take that 👍 
Thanks so much for your prompt help!!
